deploy/clowdapp.yaml

```diff
@@ -863,6 +896,18 @@ parameters:
 - name: MEMORY_LIMIT_NOTIFICATOR
   description: Maxmimum memory limit for pod
   value: "512Mi"
+- name: CPU_LIMIT_EXPLOIT_SYNC
+  description: Maximum CPU limit for pod
+  value: 500m
+- name: MEMORY_LIMIT_EXPLOIT_SYNC
+  description: Maximum CPU limit for pod
+  value: "512Mi"
+- name: CPU_REQUEST_EXPLOIT_SYNC
+  description: Requested CPU for pod
+  value: 200m
+- name: MEMORY_REQUEST_EXPLOIT_SYNC
+  description: Requested memory for pod
+  value: "128Mi"
```

Comment:
memory_limit can be max 2*memory_request
```
3m19s       Warning   FailedCreate                 job/manual-exploit-sync                                                     (combined from similar events): Error creating: pods "manual-exploit-sync-4qt68" is forbidden: memory max limit to request ratio per Pod is 2, but provided ratio is 4.000000
```

Reply:
done